requirements-test.txt

```diff
@@ -1 +1,2 @@
 flake8==3.7.9
+hypothesis==4.57.0
```

Comment:
How about relaxed requirements?

```
hypothesis >= 4.57, == 4.*
```

It seems, we don't even need [environment markers](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers) to support Python 2.7 and 3.* both.

Reply:
```
--- a/requirements-test.txt
+++ b/requirements-test.txt
@@ -1 +1,2 @@
 flake8==3.7.9
+hypothesis==4.*
```

Reply:
Seems legit, nice!